main.go

```diff
@@ -52,80 +52,123 @@ var (
 	reportFile       = "report.txt"
 	dbNameOverride   string
 	instanceOverride string
+	projectOverride  string
 	filePrefix       = ""
 	verbose          bool
 )
 
+func init() {
+	flag.StringVar(&dbNameOverride, "dbname", "", "dbname: name to use for Spanner db")
+	flag.StringVar(&instanceOverride, "instance", "", "instance: Spanner instance to use")
+	flag.StringVar(&projectOverride, "project", "", "project: Google Cloud project to use")
```

Comment:
Why do we need projectOverride?
Can we use existing GCLOUD_PROJECT environment variable?

Background: there was a conscious push to limit the number of flags for HarbourBridge, especially when there were standard environment variables for the same thing.